core/services/ocr2/plugins/functions/integration_tests/v1/internal/testutils.go

```diff
@@ -488,6 +488,7 @@ func AddOCR2Job(t *testing.T, app *cltest.TestApplication, contractAddress commo
 		[pluginConfig]
 		donId = "%s"
 		contractVersion = 1
+		processOracleEventsCheckFrequencySec = 12
```

Comment:
Average Eth block time

Reply:
for tests this should be very low, 1 sec. I'd actually change this param to milliseconds so we have an option to go lower in the future.